catboost/tutorials/apply_model/catboost_rust_prediction_tutorial/train_model.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>We support string class_names, like</p><p>```cb.CatBoostClassifier(class_names=class_names) ```</p>

 _Reply via <a href='https://app.reviewnb.com/catboost/catboost/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Fixed